src/aspire/source/simulation.py

```diff
@@ -42,10 +50,31 @@ def __init__(
         noise_filter=None,
     ):
         """
-        A Cryo-EM simulation
-        Other than the base class attributes, it has:
-
-        :param angles: A n-by-3 array of rotation angles
+        A `Simulation` object that supplies images along with other parameters for image manipulation.
+
+        :param L: Resolution of projection images (integer).
+            Default is 8 or vols.resolution if a volume is provided.
+        :param n: The number of images to generate (integer).
+        :param vols: A Volume object. Default is a LegacyVolume.
+        :param states: A 1d array of n integers in the interval [0, C). The i'th integer indicates
+            the volume used to produce the i'th projection image. Default is a random set.
+        :param unique_filters: A list of Filter objects to be applied to projection images.
+        :param filter_indices: A 1d array of n integers indicating the unique_filter indices associated
+            with each image. Default is a random set of filter indices.
+        :param offsets: A n-by-2 array of coordinates to offset the images. Default is a normally
+            distributed set of offsets.
+        :param amplitude: A 1d array of n amplitudes to scale the projection images. Default is
+            a random set in the interval [2/3, 3/2].
+        :param dtype: dtype for the Simulation
+        :param C: Number of Volumes used to generate projection images. If no Volume is provided
```

Comment:
This is mutually exclusive with actually providing `vols` right? As in you'd only want to set this option if grabbing the default legacy volumes

Reply:
That's correct, but I'm noticing that `C=2` when `LegacyVolume` is called. So, something is wrong here. I'll have to look into this a little more. Thanks for the question!

Reply:
Updated the description here to make more clear. Also, had to clean up the default `LegacyVolume` to use whatever `C` is provided.